front/app/components/UI/Error/index.tsx

```diff
@@ -246,7 +246,7 @@ const Error = (props: Props) => {
           {showIcon && <ErrorIcon name="error" data-testid="error-icon" />}
 
           <ErrorMessageText data-testid="error-message-text">
-            {text && <p>{text}</p>}
+            {typeof text === 'string' ? <Box py="16px">{text}</Box> : text}
```

Comment:
I'm just wondering in which case the text does not have a string type?

Reply:
The problem is that the text can also be `JSX` element which means we are breaking some HTML rules in that case - if we try to put a `p` in a `p` for example. In this way we ensure that if it's a string it's inside a `p` and if not - we have a Box with appropriate padding. 

Writing this, I see I forgot to add the `p` to the string case so doing it now 👍